starter/source/airbag/read_monvol.F

```diff
@@ -75,7 +75,7 @@ SUBROUTINE READ_MONVOL(T_MONVOL, T_MONVOL_METADATA, ITAB, ITABM1, IPM, IGEO,
 C     NVOLU
 #include      "com04_c.inc"
 C     KMONVO, IREC
-#include      "scr17_c.inc"
+#include      "scr17_c.inc" ! LNOPT1
```

Comment:
There should be no comments after preprocessor directives. Some compilers will complain about the Fortran-like comments after the C preprocessor directive.

Reply:
Thanks,
Remove those